openedx_proversity_reports/reports/backend/base.py

```diff
@@ -0,0 +1,135 @@
+"""
+Module that contains the report backend base class.
+"""
+from celery import task
+from django.conf import settings
+from rest_framework import status
+from rest_framework.reverse import reverse
+
+from openedx_proversity_reports.edxapp_wrapper.get_student_library import \
```

Comment:
same suggestion as above, let's try to avoid imports with "/". I think it's cleaner and also increases readability.

Reply:
Addressed. @diegomillan